### PR TITLE
Update manage-creation-of-groups.md

### PR DESCRIPTION
**Pull request by revaldiv**
At line 163 added an Important note so that user knows to connect to correct environment else they will get errors (GCCH/DOD particularly).

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/create-groups/manage-creation-of-groups.md [hunk 1]
```diff
@@ -160,6 +160,9 @@ Run the script by typing:
 
 and sign in with your administrator account when prompted.
 
+> [!IMPORTANT]
+> Be sure to connect to the correct Cloud instance. For more information please refer here:https://docs.microsoft.com/en-us/office365/enterprise/powershell/connect-to-office-365-powershell#step-2-connect-to-azure-ad-for-your-office-365-subscription
```
Comment (JohanFreelancer9) [inline Office365-Admin/create-groups/manage-creation-of-groups.md]:
```suggestion
> Be sure to connect to the correct Cloud instance. For more information, please refer to [Connect to Azure AD for your Office 365 subscription](https://docs.microsoft.com/en-us/office365/enterprise/powershell/connect-to-office-365-powershell#step-2-connect-to-azure-ad-for-your-office-365-subscription). 
```

**Comment by mypil** [comment]
@revaldiv - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by revaldiv** [comment]
@mypil  i agre with JohanFreelancer9 changes - but not seeing the merge for some reason.  Might need help...

**Comment by mypil** [comment]
@MikePlumleyMSFT - PR has been copy edited and is ready for final review. 

Thank you.

cc: @kenwith

**Pull request merged**